cypress/e2e/moviepages.cy.js

```diff
@@ -0,0 +1,23 @@
+/// <reference types="cypress" />
+
+import { loadMovies } from "../../src/movies.js";
+const url = "http://localhost:5080/";
+
+it("Check movie details and DOM elements are loaded", async () => {
+	const movieList = await loadMovies();
+
+	movieList.data.forEach((movie) => {
```

Comment:
Avoid having logic in tests. What happens if the `loadMovies()` function returns an empty array? Then this loop will never run, and the test will pass even though it might be a bug.